testpb/1.pb.go

```diff
@@ -110,21 +110,85 @@ func (x *A) String() string {
 	return protoimpl.X.MessageStringOf(x)
 }
 
-func (*A) ProtoMessage() {}
+func (*A) ProtoMessage()	{}
 
-func (x *A) ProtoReflect() protoreflect.Message {
-	mi := &file_testpb_1_proto_msgTypes[0]
-	if protoimpl.UnsafeEnabled && x != nil {
-		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
-		if ms.LoadMessageInfo() == nil {
-			ms.StoreMessageInfo(mi)
+// Deprecated: Use A.ProtoReflect.Descriptor instead.
+
+// Deprecated: Use B.ProtoReflect.Descriptor instead.
+
+// 0: Enumeration
+// 1: A
+// 2: B
+// 3: A.MAPEntry
+
+// 0: A.enum:type_name -> Enumeration
+// 1: A.MESSAGE:type_name -> B
+// 2: A.MAP:type_name -> A.MAPEntry
+// 3: A.LIST:type_name -> B
+// 4: A.ONEOF_B:type_name -> B
+// 5: A.LIST_ENUM:type_name -> Enumeration
+// 6: A.MAPEntry.value:type_name -> B
+// [7:7] is the sub-list for method output_type
+// [7:7] is the sub-list for method input_type
+// [7:7] is the sub-list for extension type_name
+// [7:7] is the sub-list for extension extendee
+// [0:7] is the sub-list for field type_name
```

Comment:
one odd side effect of the AST patcher is that comments get shifted. I think the solution to this is to loop through the AST's comments and offset them by the amount of lines ProtoReflect took up.